source/_integrations/proxmoxve.markdown

```diff
@@ -100,9 +101,39 @@ proxmoxve:
 
 ## Binary Sensor
 
-The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different.
+The integration will automatically create a binary sensor for each tracked virtual machine or container. The binary sensor will either be on if the VM's state is running or off if the VM's state is different. There are als additional attributes added to each binary sensor for the node running on, VM name, VM id, VM type, CPU count and memory size.
+
+The created sensor will be called `binary_sensor.NODE_VMNAME_running`.
```

Comment:
This should not be needed, as they show up as devices?

Reply:
for now these are no devices, but simple binary sensors with attributes.